nautilus/src/nautilus/tracing/phases/SSACreationPhase.hpp

```diff
@@ -54,7 +54,7 @@ class SSACreationPhase {
 		 * @param operationIndex the operation index, which accesses the ValueRef
 		 * @return true if Value Ref is defined locally.
 		 */
-		bool isLocalValueRef(Block& block, value_ref& valRef, Type ref_type, uint32_t operationIndex);
+		static bool isLocalValueRef(Block& block, value_ref& valRef, Type ref_type, uint32_t operationIndex);
```

Comment:
Why is this function static?

Reply:
Because, it does not use any members. `Clang-Tidy` gave me the hint. If you want, I can revert this change.